.github/workflows/pr.yml

```diff
@@ -31,11 +31,24 @@ jobs:
           docker run --volume ${PWD}:/app --rm node:20 yarn --cwd /app prettier src --check
 
   audit:
-    name: yarn audit
+    name: audit
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: yarn audit
-        run: yarn audit
+      - name: npm audit
+        run: npm audit
+
+  check-create-release:
+    runs-on: ubuntu-latest
+    name: Test create release
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Create test release
+        run: |
+          ./bin/create-release 0.0.0
```

Comment:
Why do you use `0.0.0` here and not something like `dev-test`?

Reply:
I'll change it. I think i did it because I was changing the version of the package.json in the create release script, and it broke if it wasnt a valid semver thingy